ai-server/app/music/setup.py

```
@@ -32,7 +32,7 @@
 except FileNotFoundError:
     long_description = DESCRIPTION
 
-REQUIRED = [i.strip() for i in open(HERE / 'requirements.txt') if not i.startswith('#')]
+REQUIRED = [i.strip() for i in open(HERE / 'requirements-cpu.txt') if not i.startswith('#')]
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)